src/sentry/monitors/consumers/monitor_consumer.py

```diff
@@ -501,13 +508,10 @@ def _process_checkin(item: CheckinItem, txn: Transaction | Span):
 
     monitor_config = params.pop("monitor_config", None)
 
-    params["duration"] = (
+    if params.get("duration") is not None:
```

Comment:
I suspect this might be what caused https://sentry.sentry.io/issues/6237719084/?project=1&query=release%3Abackend%401e5d5a562bbc4789c15023775b50f1c0713fc5f0&referrer=release-issue-stream

Reply:
hmm I'm not so sure -- the error is before that line 🤔 this might be the schemas update I made?  though afaict it should be fine with `string` there

Reply:
yeah this was the schemas bump instead and actually not this patch. suspect commits blamed this before it had even gone out